{"name": "testcaseadd3[【类型】输入特殊字符，给出提示信息1]", "status": "passed", "steps": [{"name": "步骤一：*&和0.2开始计算，预期应等于参数大小超出范围，打印<ExceptionInfo for raises contextmanager>", "status": "broken", "statusDetails": {"message": "TypeError: '>' not supported between instances of 'str' and 'int'\n", "trace": "  File \"D:\\zj\\PycharmProjects\\study202208\\practice\\study\\study0824\\testcase\\test_add.py\", line 48, in testcaseadd3\n    result = self.cal.add(x,y)\n  File \"D:\\zj\\PycharmProjects\\study202208\\script\\calculator.py\", line 4, in add\n    if a > 99 or a < -99 or b > 99 or b < -99:\n"}, "start": 1661497761876, "stop": 1661497761876}], "attachments": [{"name": "log", "source": "98aaa9aa-22d2-40a9-8d2e-a874f502babd-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "x", "value": "'*&'"}, {"name": "y", "value": "0.2"}, {"name": "expect", "value": "'参数大小超出范围'"}], "start": 1661497761875, "stop": 1661497761876, "uuid": "b2ef0757-9e2a-48e2-bfef-2b38232a131a", "historyId": "6c9ab57b6fd1834943a6ca40aa60f8ea", "testCaseId": "2f8f937ca82ff540c8b5f46a0207716d", "fullName": "practice.study.study0824.testcase.test_add.Testadd#testcaseadd3", "labels": [{"name": "story", "value": "加法模块测试error"}, {"name": "tag", "value": "P2"}, {"name": "parentSuite", "value": "practice.study.study0824.testcase"}, {"name": "suite", "value": "test_add"}, {"name": "subSuite", "value": "Testadd"}, {"name": "host", "value": "LAPTOP-BEO6NH7O"}, {"name": "thread", "value": "33120-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "practice.study.study0824.testcase.test_add"}]}